scripts/release.js

```diff
@@ -59,12 +59,12 @@ async function doRelease() {
     const releaseOptions = {
       increment: isPreRelease ? 'prerelease' : versionNumber,
       preReleaseId: preReleaseTag,
-      // ci: true,
+      ci: false,
```

Comment:
Commented out by mistake. It was set to `true` to avoid the questions being asked in the command line, but this doesn't allow providing the 2fa code for npm publishing, so setting it to `false`